spyder_terminal/config.py

```diff
@@ -25,14 +25,14 @@
      ),
     ('shortcuts',
      {
-      'terminal/copy': 'Ctrl+Alt+Shift+C' if LINUX else 'Ctrl+Alt+C',
-      'terminal/paste': 'Ctrl+Alt+Shift+V' if LINUX else 'Ctrl+Alt+V',
-      'terminal/new terminal': 'Ctrl+Alt+T',
-      'terminal/clear': 'Ctrl+Alt+K',
-      'terminal/zoom in': 'Ctrl++',
-      'terminal/zoom out': 'Ctrl+-',
+       'terminal/copy': 'Ctrl+Alt+Shift+C' if LINUX else 'Ctrl+Alt+C',
+       'terminal/paste': 'Ctrl+Alt+Shift+V' if LINUX else 'Ctrl+Alt+V',
+       'terminal/new_terminal_cwd_action': 'Ctrl+Alt+T',
```

Comment:
Why is this shortcut associated to `new_terminal_cwd_action` and not to `new_terminal`?

Reply:
By default `new_terminal` will create the new one on the current cwd. We divide the two of them because one was linked to the button on the toolbar, while the other is linked in the hamburger menu.

Reply:
And the one registered with the shortcut is the `new_terminal_cwd`